src/librustc_data_structures/fingerprint.rs

```diff
@@ -9,6 +9,7 @@
 // except according to those terms.
```

Comment:
Can we not move `Fingerprint` to `rustc_datastructures`? It's an incremental compilation specific type.